tlatools/org.lamport.tlatools/test/tlc2/debug/GetScopedIdentifiersTests.java

```diff
@@ -0,0 +1,104 @@
+package tlc2.debug;
```

Comment:
Copyright header

Reply:
Do we really need the MIT header in every file?

Reply:
IANAL, but this is standard practice, and most IDEs typically fold the license header automatically.